README.md

```diff
@@ -345,12 +345,7 @@ clasp <command>
 
 #### Run Tests (experimental)
 
-Change `describe.skip(...)` to `describe(...)` for relevant tests.
-
-```sh
-sudo npm run build;
-npm run test
-```
+See `/tests/README.md`
```

Comment:
Write a tl;dr for how to run tests.

I'd keep:

```sh	
sudo npm run build;	
npm run test	
```

Reply:
I don't see this in the tests README.

Reply:
It's under the `Configuration Locally` header.